content/html/concepts/elements/terms/del/del.md

```diff
@@ -13,7 +13,7 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-The **`<del>`** element is used to show a deleted portion of text and is commonly intended to show the changes made to a document while keeping a history of the changes made. Browsers render it showing strike-through text.
+The **`<del>`** element is used to show a deleted portion of text and is commonly intended to show the changes made to a document while keeping a history of the changes made. Browsers render it showing strike-through text: ~~example~~
```

Comment:
can we also mention that the style could be changed?
```suggestion
The **`<del>`** element is used to show a deleted portion of text and is commonly intended to show the changes made to a document while keeping a history of the changes made. Though the style can be changed using [CSS](https://www.codecademy.com/resources/docs/css), browsers usually render it as strike-through text: ~~example~~
```

Reply:
I liked you idea but instead of adding a link to CSS pages I have added a direct link to to the page that tells how to acheive strike through text with CSS.

Reply:
@ETurner-Bisset That works too. Let me know when you add the change, please!

Reply:
@KTom101 i am pretty sure I have  pushed the changes. Let me know if I have done something wrong.